packages/webapp/src/pages/election/round-video-upload.tsx

```diff
@@ -196,43 +169,44 @@ const RoundVideoUploadList = ({ election }: { election: CurrentElection }) => {
                 videoFile
             );
 
-            roundVideoList[roundIndex] = videoHash;
-            setRoundVideoList(roundVideoList);
-
             setVideoSubmissionPhase(undefined);
-            setUploadCompleteMessage("Election video uploaded successfully!");
+            setUploadCompleteMessage({
+                roundIndex,
+                message: "Election video uploaded successfully!",
+            });
         } catch (error) {
             onError(error, "Unable to set the election round video");
```

Comment:
question: my IDE has started complaining to me about these errors:

![image](https://user-images.githubusercontent.com/7911424/134529258-25fabbc1-1eb9-4788-bf4a-503fddd2daea.png)

Any idea why? It stops complaining if I do: `onError(error as Error, "Unable to set the ele...")`

Is that happening for you too?

Reply:
Man, I started with a fresh environment today and I started to receive this!

Reply:
(It was not happening before)